subsys/shell/Kconfig.backends

```diff
@@ -15,7 +15,7 @@ menuconfig SHELL_BACKENDS
 if SHELL_BACKENDS
 
 config SHELL_BACKEND_SERIAL
-	bool "Enable serial backends."
+	bool "Enable serial backends"
```

Comment:
since you fixed that you could remove `s` and the end of backends.

Reply:
done